### DISCUSSION TITLE
How do we dismiss the Github Organizations "Onboarding Guide"

### DISCUSSION THREAD

**Discussion by tedivm** [community/community#124565] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I can't get this page to go away. I've done all the tasks, let it create a new demo repository and jump through hoops for things I've known how to do for over a decade, and the tasks just keep resetting. Is there a way to get rid of this?

**Comment by tedivm** [comment]
This is what I'm referring to:

<img width="940" alt="Screenshot 2024-05-18 at 12 58 10 PM" src="https://github.com/community/community/assets/121709/191c87d7-d1e4-4a05-ad53-eb26102bf91f">

**Comment by rich-fs** [comment, marked as answer]
There's a link on the side ... "Hide the tasks we've suggested" ... d/w took me ages to find it, even went to the trouble of completing all the tasks in the hope that would make it go away ... spoiler it didn't!

![image](https://github.com/community/community/assets/120682131/b998633d-db7a-4221-bc29-8ba9bd4e7388)

**Reply by OlivierJM** [reply]
Very helpful, Thanks

**Reply by zero-evoke** [reply]
the worst possible ux with that link. if i had to gogle to get here to find that link  - is a bad user experience and bad design. i had to find that option in intuitive way

**Reply by willhackett** [reply]
Thank you - has been driving me mental.

**Comment by Edgarruiz8585** [comment]
@github by Edgarruiz8585

**Comment by mohanidragel** [comment]
I'm delighted we are able to hide it. This page is the worst addition to humanity since covid.
